### ISSUE TITLE
Mudar a categorização de página para a página de discussão

### ISSUE THREAD

**Issue by themudo** [WikiMovimentoBrasil/wikiconcursos#102]
Conforme discussão em https://pt.wikipedia.org/wiki/Wikip%C3%A9dia:Esplanada/propostas/Recomenda%C3%A7%C3%A3o_para_que_a_categoriza%C3%A7%C3%A3o_de_artigos_no_escopo_de_projetos_seja_feita_nas_p%C3%A1ginas_de_discuss%C3%A3o_desses_artigos_(12out2024), a categorização das páginas deve ser feito na página de discussão dos artigos, não nas páginas do domínio principal.

**Comment by rovertrack** [comment]
@themudo  is this ticket still valid, can i work on this?

**Comment by themudo** [comment]
Yes, I think it is. I am not project owner, so I can't really say.
